packages/core/src/components/essentials/tooltip.ts

```diff
@@ -207,7 +222,40 @@ export class Tooltip extends Component {
 	}
 
 	render() {
-		this.tooltip.classed("hidden", true);
+		const isTooltipEnabled = Tools.getProperty(
+			this.model.getOptions(),
+			"tooltip",
+			"enabled"
+		);
+		if (isTooltipEnabled) {
+			// Grab the tooltip element
+			const holder = select(this.services.domUtils.getHolder());
+			const chartprefix = Tools.getProperty(
+				this.model.getOptions(),
+				"style",
+				"prefix"
+			);
+			this.tooltip = DOMUtils.appendOrSelect(
+				holder,
+				`div.${settings.prefix}--${chartprefix}--tooltip`
+			);
+
+			// Apply html content to the tooltip
+			const tooltipTextContainer = DOMUtils.appendOrSelect(
+				this.tooltip,
+				"div.content-box"
+			);
```

Comment:
do we need this query in here? would it not being there break anything? seems like it's not used